.github/workflows/backend.yml

```diff
@@ -313,7 +313,7 @@ jobs:
       - name: run updated modules integration test (part-1)
         if: needs.changes.outputs.api == 'false' && needs.changes.outputs.it-modules != ''
         run: |
-          sub_modules=`python tools/update_modules_check/update_modules_check.py sub_update_it_module ${{needs.changes.outputs.it-modules}} 8 0`
+          sub_modules=`python tools/update_modules_check/update_modules_check.py sub_update_it_module ${{needs.changes.outputs.it-modules}} 10 0`
```

Comment:
why add new `update_modules_check` module? I think 8 modules are enough for now.

Reply:
The CI subtask is interrupted due to timeout.   
https://github.com/hawk9821/seatunnel/actions/runs/9689364040/job/26756996209